topic04-git-introduction/book-git-tutorial/indexchat.html

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
not sure about this file? seems to set an `Open Chat` button which just opens an empty section

Reply:
@laurafitzgerald That's actually in all sections. It's a generated file. We can remove it if needed. It seems to direct to gitter

Reply:
@JameelB I think we should remove it as it opens up only an empty panel.